proto/astarteplatform/msghub/astarte_message.proto

```diff
@@ -26,14 +26,16 @@ import "google/protobuf/timestamp.proto";
 
 import "astarteplatform/msghub/astarte_type.proto";
 
+/* Astarte message to be used when sending data to Astarte. */
 message AstarteMessage{
-  string interface_name = 1;
-  string path = 2;
-  oneof payload {
-    AstarteDataType astarte_data = 3;
-    AstarteUnset astarte_unset = 4;
+  string interface_name = 1; // Name of the interface to send data on.
+  string path = 2;           // Endpoint to send the data on.
+  oneof payload {            // Content of the message.
+    AstarteDataType astarte_data = 3; // Effective payload.
+    AstarteUnset astarte_unset = 4;   // Null payload.
```

Comment:
Choose whether to leave a single space after the semicolon or to line it up with the other comments.